src/snacks/SnackTags.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import tags from './tags.json'
+import PropTypes from 'prop-types'
+
+SnackTags.propTypes = {
+  currentSnack: PropTypes.number.isRequired
+}
+
+export default function SnackTags({ currentSnack }) {
+  function showTags() {
```

Comment:
Der Code sieht gut aus, aber die Namensgebung ist hier etwas verwirrend. Die showTags-Komponente könntest du auch einfach weglassen und die Tags direkt in die Liste mappen.

Reply:
top, danke!